src/app/backend/scaling/scale.go

```diff
@@ -46,9 +47,10 @@ func GetScaleSpec(client client.Interface, kind, namespace, name string) (rc *Re
 // method since the client scale method does not provide one for the job.
 func ScaleResource(client client.Interface, kind, namespace, name, count string) (rc *ReplicaCounts, err error) {
 	rc = new(ReplicaCounts)
-	if strings.ToLower(kind) == "job" {
+        kind = strings.Replace(kind, " ", "", -1)
```

Comment:
This is not a proper fix. You should fix it in frontend. Only changing hardcoded strings is ok.